transaction-status/src/parse_address_lookup_table.rs

```diff
@@ -0,0 +1,268 @@
+use {
+    crate::parse_instruction::{
+        check_num_accounts, ParsableProgram, ParseInstructionError, ParsedInstructionEnum,
+    },
+    bincode::deserialize,
+    serde_json::json,
+    solana_address_lookup_table_program::instruction::ProgramInstruction,
+    solana_sdk::{instruction::CompiledInstruction, message::AccountKeys},
+};
+
+pub fn parse_address_lookup_table(
+    instruction: &CompiledInstruction,
+    account_keys: &AccountKeys,
+) -> Result<ParsedInstructionEnum, ParseInstructionError> {
+    let address_lookup_table_instruction: ProgramInstruction = deserialize(&instruction.data)
+        .map_err(|_| {
+            ParseInstructionError::InstructionNotParsable(ParsableProgram::AddressLookupTable)
+        })?;
+    match instruction.accounts.iter().max() {
+        Some(index) if (*index as usize) < account_keys.len() => {}
+        _ => {
+            // Runtime should prevent this from ever happening
+            return Err(ParseInstructionError::InstructionKeyMismatch(
+                ParsableProgram::AddressLookupTable,
+            ));
+        }
+    }
+    match address_lookup_table_instruction {
+        ProgramInstruction::CreateLookupTable {
+            recent_slot,
+            bump_seed,
+        } => {
+            check_num_address_lookup_table_accounts(&instruction.accounts, 4)?;
+            Ok(ParsedInstructionEnum {
+                instruction_type: "createLookupTable".to_string(),
+                info: json!({
+                    "lookupTableAccount": account_keys[instruction.accounts[0] as usize].to_string(),
+                    "lookupTableAuthority": account_keys[instruction.accounts[1] as usize].to_string(),
+                    "source": account_keys[instruction.accounts[2] as usize].to_string(),
```

Comment:
```suggestion
                    "payerAccount": account_keys[instruction.accounts[2] as usize].to_string(),
```

Reply:
I chose these labels to be consistent with all the other programs we offer parsing for. If you want them to diverge, please make the case.
Also, on the topic of inconsistency, it's annoying that the lookup-table instruction builders take `Pubkey`s, while the builders for all the other native programs take `&Pubkey`s

Reply:
Oh I see, I copied this name from the parsed bpf loader instructions. I guess we already have some inconsistency here. As for the `Pubkey` vs `&Pubkey` issue, I'll put up a patch, thanks for pointing that out!

Reply:
Welp, pre-existing inconsistency is a good argument. Alas 😢  
I guess I've no strong opinion which we match, then. Do you prefer `payerAccount`/`recipient` to `source`/`destination`?

Reply:
Done